CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+# 10.2.0-rc.0 "cotton-cactus" (2020-08-26)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| feature |  add support for TypeScript 4.0 ([#20368](https://github.com/angular/components/issues/20368)) ([f33210c](https://github.com/angular/components/commit/f33210c)) |
```

Comment:
Should this be part of highlights?

Reply:
I'll add a highlights section for the stable release